tirgei/Questioner-API-V2
# #163221842 Setup config environments

#### What does this PR do?
This Pull Request creates the config environments for the app which allows for different running configurations

#### Description of Task to be completed?
The tasks for this PR is:
- Write tests for different running environments
- Create config file and add config environments
- Create function to initialize Flask app

#### How should this be manually tested?
Clone this repository on your machine:
```$ git clone https://github.com/tirgei/Questioner-API.git ```

CD into the project folder on your machine
```$ cd Questioner-API ```

Check out the branch for this PR
``` $ git checkout  ch-setup-config-163221842```

Run tests
``` $ pytest --verbose```


#### Pivotal Tracker story associated with this PR
[#163221842](https://www.pivotaltracker.com/story/show/163221842)

#### Screenshots
Failing test
![test-config-fail](https://user-images.githubusercontent.com/17829576/51200066-8186b580-1909-11e9-9a35-410893a128fd.png)

Passing test
![test-config-pass](https://user-images.githubusercontent.com/17829576/51200292-2903e800-190a-11e9-991f-c3e44bcc30c6.png)
